frontend/src/employee-frontend/components/vasu/sections/VasuEvents.tsx

```diff
@@ -74,8 +65,16 @@ export function VasuEvents({
         <ChipContainer>
           <VasuStateChip state={documentState} labels={i18n.vasu.states} />
         </ChipContainer>
-        <EventRow label={i18n.vasu.lastModified} date={modifiedAt} />
-        <EventRow label={i18n.vasu.lastPublished} date={lastPublished} />
+        <EventRow
+          label={i18n.vasu.lastModified}
+          date={LocalDate.fromSystemTzDate(modifiedAt)}
+        />
+        <EventRow
+          label={i18n.vasu.lastPublished}
+          date={
+            lastPublished ? LocalDate.fromSystemTzDate(lastPublished) : null
```

Comment:
These conversions are now done on each render. Eg. on every keystroke in a text input. It might have a (maybe negligible) performance impact when there are multiple events.
Should we just convert all event dates to `LocalDate` in response mapping? I cannot think of a feature where timestamps are required.

Reply:
Hmm yeah, in figma it only shows dates at the moment, but I was actually wondering if the publishing should also include time. Sometimes you might publish multiple times within a day, e.g. fix some typo in between. But maybe your idea is best for now.

Reply:
Or just disregard the potential perf issue for now 😄